pkg/volume/util/atomic_writer.go

```diff
@@ -193,6 +195,14 @@ func (w *AtomicWriter) Write(payload map[string]FileProjection) error {
 	}
 
 	// (8)
+	if setPerms != nil {
+		if err := setPerms(); err != nil {
+			klog.Errorf("%s: error applying volume ownership settings: %v", w.logContext, err)
```

Comment:
drop "volume" here - it's minor but this makes it a smidge more general purpose

Reply:
Fixed!